apps/web/src/Pages/Staking/Lockup/Lockup.tsx

```
@@ -297,30 +425,41 @@ function Lockup({
+                                availableBalance > txnCost &&
+                                i === Number(period) &&
+                                !error
+                              }
                               hover={true}
-                              onClick={() => setPeriod(i.toString())}
+                              onClick={() => {
+                                if (availableBalance <= txnCost || error)
+                                  return;
+                                setPeriod(i.toString());
+                              }}
                             >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)